tests/test_analyze.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+from .doppel.cil.bin import analyze as al
+
+
+class AnalyzeTest(unittest.TestCase):
+    
+   	#Test for _log_info and should print msg
+	def test_log_info(self):
+
+		self.assertEqual(al._log_info("Hello World!"), "Hello World", "Printed the msg")
+
+
+	#Test for _get_arg_names and should return arg names.
```

Comment:
it looks like you started this but didn't finish it. Can you please remove this commented-out code?